tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements.txt

```diff
@@ -0,0 +1,15 @@
+--pre
+-f https://download.pytorch.org/whl/nightly/cu101/torch_nightly.html
+# transformers requires sklearn
+pandas
+sklearn
+numpy==1.19.5
```

Comment:
If you use numpy 1.19, then the built package can't be used with numpy 1.16 and basically it is not distributable. It's fine if it is only used for CI build testing. But when your team decided to publish the package, you need to change this part.

Reply:
This change is only for the CI build. It does not impact the packaging pipeline and should not impact our distribution.